cli/packaging/ubuntu/postinst

```diff
@@ -12,19 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+create_template_dir() {
+    if [ ! -d $SPLINTER_CIRCUIT_TEMPLATE_PATH ]; then
+        mkdir -p $SPLINTER_CIRCUIT_TEMPLATE_PATH
+    fi
+}
+
+prepare_template_dir() {
+    chmod 755 $SPLINTER_CIRCUIT_TEMPLATE_PATH
+    chmod 640 $SPLINTER_CIRCUIT_TEMPLATE_PATH/*.yaml
+}
+
 case "$1" in
     configure)
-
     # This value is present only during upgrades
     if [ -z "$2" ]; then
-        ln -s /usr/bin/splinter /usr/bin/splinter-cli
-
-        if [ ! -d /usr/share/splinter/circuit-templates ]; then
-          mkdir -p /usr/share/splinter/circuit-templates
+        if [ -z $SPLINTER_CIRCUIT_TEMPLATE_PATH ]
```

Comment:
This script should probably ignore SPLINTER_CIRCUIT_TEMPLATE_PATH and always install into the well-known directory; in any case, this code wouldn't work if SPLINTER_CIRCUIT_TEMPLATE_PATH had more than one directory specified.

Reply:
That's fair, the script was initially meant to capture the environment variable assuming that the default directory wouldn't be used. Now, with the multiple directories, the default is always searched last so this commit is unnecessary - I'll remove it.